ISPP-Eventia/eventus
# Feature/#52 landing page

close #52

  ## changes
  - add basic frontend configurations and structures for the app
  - add colors and animations to the theme config files
  - implement the landing page
  - implement a basic header
  - implement a basic footer with contact info

  ## steps to review
- install frontend new dependencies `npm install`
- run the frontend `npm start`
- check the frontend landing page
- check the code and configurations

## concerns
- the color schema is not clearly defined yet
